src/core/libraries/libscegnmdriver.cpp

```diff
@@ -2,1264 +2,1264 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 
 // Generated By moduleGenerator
-#include "common/log.h"
+#include "common/logging/log.h"
 #include "error_codes.h"
 #include "libscegnmdriver.h"
 
 namespace Libraries::GnmDriver {
 
 int PS4_SYSV_ABI sceGnmAddEqEvent() {
-    PRINT_UNIMPLEMENTED_FUNCTION_NAME();
+    LOG_WARNING(Lib_GnmDriver, "(STUBBED) called");
```

Comment:
hmm why changed? the previous was printing the function name , now everyone gets the same output

Reply:
also why warning? most of the times that call might be fatal  for emulation :D

Reply:
Can make it into an error yeah. If you have suggestion on what unimplemented function format should look, open to suggestions too